src/js/components/Breadcrumb.js

```diff
@@ -1,13 +1,17 @@
 import classNames from "classnames";
 import PropTypes from "prop-types";
 import React from "react";
+import isEqual from "lodash.isequal";
 
 class Breadcrumb extends React.Component {
   shouldComponentUpdate(nextProps) {
     return (
       this.props.isCaret !== nextProps.isCaret ||
       this.props.isIcon !== nextProps.isIcon ||
-      this.props.title !== nextProps.title
+      this.props.title !== nextProps.title ||
+      this.props.instancesCount !== nextProps.instancesCount ||
```

Comment:
Would it be possible to do these `shouldComponentUpdate` checks in `ServiceBreadcrumb` instead?

Reply:
We do them there, but we also have to do them here.